UPGRADE-3.4.md

```diff
@@ -68,6 +68,9 @@ Process
 
  * The `Symfony\Component\Process\ProcessBuilder` class has been deprecated,
    use the `Symfony\Component\Process\Process` class directly instead.
+   
+ * The `Symfony\Component\Process\Process` __construct `cwd` argument will throw 
+   a deprecation warning on `run` if the folder does not exist.
```

Comment:
this should target `3.4`. The upgrade note should be added when deprecating the behavior, we take care of doing the change when merging it up to master (because master is 4.0, where we remove BC layers).